### ISSUE TITLE
Missing keyframes when combining multiple animations

### ISSUE THREAD

**Issue by mohitsinghs** [tailwindlabs/tailwindcss#4711]
### What version of Tailwind CSS are you using?

2.2.0

### What build tool (or framework if it abstracts the build tool) are you using?

None

### What version of Node.js are you using?

None

### What browser are you using?

Firefox

### What operating system are you using?

Ubuntu 20.04

### Reproduction repository

https://play.tailwindcss.com/clI0hhAxuo

### Describe your issue

When combining multiple animations ( for example `animation-1, animation-2-after-delay`  ) while extending animations and keyframes, the keyframes related to combined animations are missing from the build.

To see this bug in action, run [this](https://play.tailwindcss.com/clI0hhAxuo) with both available versions of tailwindcss on play.tailwindcss.com. The older version correctly includes all keyframes and both logo and bullets are animated but when switching to v2.2.0, logo has no animation since keyframes related to those animations are not included.

**Comment by mohitsinghs** [comment]
⌛ The workaround for now is to define keyframes for combined animations in css instead of extending theme.

**Comment by mohitsinghs** [comment]
So, I did some digging and spotted the culprit.
https://github.com/tailwindlabs/tailwindcss/blob/0413b84c12e44950b84b144fafaffb5266f371b8/src/plugins/animation.js#L23
for combined animations parseAnimationValue returns array of parsed animations which causes animationName to be null.
@adamwathan is it intentional ? If not, I wrote some tests and a simple fix which deals with combined animations differently. The tests are passing and I'll be happy to send a pr.
